cedar-policy-validator/src/entity_manifest/loader.rs

```diff
@@ -435,7 +435,7 @@ fn compute_ancestors_request(
 
     while !to_visit.is_empty() {
         let mut next_to_visit = vec![];
-        for entity_request in to_visit.drain(..) {
+        for entity_request in to_visit.into_iter() {
```

Comment:
I think `.into_iter()` is implicit here, so this could just be `for entity_request in to_visit`?